contracts/utils/cryptography/P256.sol

```diff
@@ -285,6 +288,9 @@ library P256 {
      * │  8 │ 2g 2g+p 2g+2p 2g+3p │
      * │ 12 │ 3g 3g+p 3g+2p 3g+3p │
      * └────┴─────────────────────┘
+     *
+     * Note that in the negligible scenario where `P` is `-G`, the addition of `points[0x05]` (`G` + `P`) will result
```

Comment:
As discussed in https://github.com/OpenZeppelin/openzeppelin-contracts/pull/5218/files#r1771415748, there are other cases then the obvious `P = G` that would cause this to fail.

Reply:
```suggestion
     * Note that because of limitation to the `_jAdd` function (which returns invalid results when adding a point
     * with itself), this function could produce invalid values. This would result in the inability to verify or recover
     * valid signatures. It is estimated to have a probability of about 1e-74 to happen.
```